src/main/kotlin/spatialcrafting/compat/rei/util/EmptyWidget.kt

```diff
@@ -4,7 +4,7 @@ import me.shedaniel.rei.gui.widget.Widget
 import net.minecraft.client.gui.Element
 
 class EmptyWidget : Widget() {
-    override fun children(): List<Element> = listOf()
+    override fun children(): List<Element> = emptyList()
```

Comment:
I think the implementation of `listOf()` is `emptyList()`

Reply:
`emptyList()` returns `EmptyList`, which is immutable & doesn't create a new list instance.

Reply:
Yes but
![image](https://user-images.githubusercontent.com/25990014/64182543-0f9bb600-ce71-11e9-9ada-02c2bafa6571.png)

Reply:
I tried